borg/archiver.py

```diff
@@ -1488,6 +1488,10 @@ def sig_info_handler(signum, stack):  # pragma: no cover
             break
 
 
+def sig_term_handler(signum, stack):
+    raise KeyboardInterrupt()
```

Comment:
I think we should be able to make a difference between SIGTERM exception and SIGINT exception.